langchain/callbacks/manager.py

```diff
@@ -200,11 +200,15 @@ def _handle_event(
                     **kwargs,
                 )
             else:
-                logger.warning(f"Error in {event_name} callback: {e}")
+                logger.warning(
+                    f"Error in {handler.__class__.__name__}.{event_name} callback: {e}"
+                )
         except Exception as e:
             if handler.raise_error:
                 raise e
-            logger.warning(f"Error in {event_name} callback: {e}")
+            logger.warning(
```

Comment:
Just thinking. Maybe moving warning before `if handler.raise_error:` adds helpful information for debugging? Maybe `raise e` would provide enough debugging info. I'm not sure.

Reply:
Yeah that's a good point, especially if it's running on a separate thread. I'll make the change